samples/attested_tls/server/enc/openssl_server.cpp

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <arpa/inet.h>
+#include <openenclave/enclave.h>
+#include <openssl/evp.h>
+#include <openssl/ssl.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include "../../common/openssl_utility.h"
+
+extern "C"
+{
+    int set_up_tls_server(char* server_port, bool keep_server_up);
+};
+
+int verify_callback(int preverify_ok, X509_STORE_CTX* ctx);
+
+int initalize_ssl_context(SSL_CTX*& ctx)
+{
+    int ret = -1;
+    if ((ctx = SSL_CTX_new(TLS_server_method())) == NULL)
+    {
+        printf(TLS_SERVER " unable to create a new SSL context\n");
+        goto exit;
+    }
+    // Exclude SSLv2, SSLv3, TLS 1.0, TLS 1.1 and TLS 1.2
+    SSL_CTX_set_options(ctx, SSL_OP_NO_SSLv2);
+    SSL_CTX_set_options(ctx, SSL_OP_NO_SSLv3);
+    SSL_CTX_set_options(ctx, SSL_OP_NO_TLSv1);
+    SSL_CTX_set_options(ctx, SSL_OP_NO_TLSv1_1);
+    SSL_CTX_set_options(ctx, SSL_OP_NO_TLSv1_2);
+    ret = 0;
+exit:
+    return ret;
+}
+
+int create_listener_socket(int port, int& server_socket)
+{
+    int ret = -1;
+    struct sockaddr_in addr;
+    addr.sin_family = AF_INET;
+    addr.sin_port = htons(port);
+    addr.sin_addr.s_addr = htonl(INADDR_ANY);
+
+    server_socket = socket(AF_INET, SOCK_STREAM, 0);
+    if (server_socket < 0)
+    {
+        printf(TLS_SERVER "socket creation failed \n");
+        goto exit;
+    }
+
+    if (bind(server_socket, (struct sockaddr*)&addr, sizeof(addr)) < 0)
+    {
+        printf(TLS_SERVER "Unable to bind socket to the port \n");
+        goto exit;
+    }
+
+    if (listen(server_socket, 20) < 0)
+    {
+        printf(TLS_SERVER "Unable to open socket for listening \n");
+        goto exit;
+    }
+    ret = 0;
+exit:
+    return ret;
+}
+
+int handle_communication_until_done(
+    int& server_socket_fd,
+    int& client_socket_fd,
+    SSL_CTX*& ssl_server_ctx,
+    SSL*& ssl_session,
+    bool keep_server_up)
+{
+    int ret = -1;
+
+waiting_for_connection_request:
+
+    // reset ssl_session setup and client_socket_fd to prepare for the new TLS
+    // connection
+    close(client_socket_fd);
+    SSL_free(ssl_session);
+    printf(TLS_SERVER " waiting for client connection \n");
```

Comment:
```suggestion
    printf(TLS_SERVER " waiting for client connection\n");
```
And same nit in many other places in this file